nexus/users/models.py

```diff
@@ -30,17 +32,45 @@ def create_superuser(self, email: str, password: str, **extra_fields):
 
 
 class User(AbstractBaseUser, UUIDModel, PermissionsMixin):
-    first_name = models.CharField(_('First Name'), max_length=120, blank=True)
-    last_name = models.CharField(_('Last Name'), max_length=120, blank=True)
+
+    GENDER_CHOICES = Choices(
+        ('NOT_SELECTED', 'not_provided', _('Not Selected')),
```

Comment:
Can you change the options for all the choices you've used?

Like you did for `not_provided`. Also, either keep everything `not_selected` or `not_provided`. Don't mismatch between code and values.